src/stylesheets/components/_blank-slate.scss

```diff
@@ -0,0 +1,15 @@
+.blank-slate {
+  text-align: center;
+  padding-top: rem(250px);
+
+  &__emoji {
+    font-size: map-get($font-sizes, 'xxl');
+  }
+
+  &__message {
+    color: map-get($brand-white, 'primary');
+    font-weight: map-get($font-weight, 'bolder');
+    font-size: map-get($font-sizes, 'xl');
```

Comment:
Should we use `font-size: rem(28px)` here ? 
![Screen Shot 2563-11-11 at 12 22 46](https://user-images.githubusercontent.com/1772999/98772228-cd78e780-2418-11eb-97b5-da644ec84fe1.png)

if the font size does not align with the declared one maybe we can have a separate variable for the display font sizes 🙌 
![Screen Shot 2563-11-11 at 12 26 04](https://user-images.githubusercontent.com/1772999/98772346-221c6280-2419-11eb-8293-01a7357a9ef1.png)